audiophiler/templates/base.html

```diff
@@ -44,6 +44,33 @@
                             My Files
                         </a>
                     </li>
+                    {% if file.author == auth_dict["uid"] or is_rtp or is_eboard %}
+                    <li class="dropdown">
+                        <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">
+                            <span class="glyphicon glyphicon-music"></span> Harold
+                            <span class="caret"></span>
+                        </a>
+                        <ul class="dropdown-menu">
+                            <li>
+                                <a href="/admin">
```

Comment:
I still feel as though this could reasonably be a separate navbar option.

Would you be able to post screenshots to show how this looks?

Reply:
Also, why not call this tour mode, since it's all about tour mode? I think that's a little more intuitive.

Reply:
I don't have a testing environment, so no. This is similar code to what was used in conditional, it was roughly what I was going for so I figured it would be fine as a stand in until I could see what it looked like.

/admin is what Nate told me to send